### PR TITLE
Add PPP and fix PE-D bugs

### LINKED ISSUE #200: [PE-D] Using Commands while recipe drawer is open
After opening a recipe drawer, using the command selectR INDEX, I am still able to use other commands, such as addR, deleteR, editR.

The commands are carried out while the recipe drawer is still open and I am unable to see the recipe list. Not sure if this is intended or is a bug as it is not stated in the UG.

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: PrestonTYR/ped#2

### LINKED ISSUE #203: [PE-D] Close command can still be called even when recipe drawer is not open
The close command can still be called when the recipe drawer is not open. 

-------------
Labels: `severity.VeryLow` `type.FeatureFlaw`
original: PrestonTYR/ped#5

### LINKED ISSUE #204: [PE-D] Recipe commands can be called while viewing the fridge and vice versa
Not sure if this is a bug, but after displaying ingredients in the fridge, recipe commands, such as editR and deleteR can still be called while the ingredients list is open.

Similarly, fridge commands, such as editF and deleteF can be called while the recipe list is open.

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: PrestonTYR/ped#6

### LINKED ISSUE #206: [PE-D] Recipe commands can be called while viewing calories
Recipe commands such as deleteR and editR can be called while viewing the consumption list/ calories

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: PrestonTYR/ped#8

### LINKED ISSUE #207: [PE-D] Fridge commands can be called while viewing calories
Fridge commands, such as deleteF and editF can be called while viewing consumption list/calories.

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: PrestonTYR/ped#9

### PR DESCRIPTION
**Pull request by daongochieu2810**

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/recipe/SelectRecipeCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.recipe.Recipe;
+
+/**
+ * Contains integration tests (interaction with the Model, UndoCommand and RedoCommand) and unit tests for
```
Comment (caitlinjee) [inline src/test/java/seedu/address/logic/commands/recipe/SelectRecipeCommandTest.java]: dont forget to update the javadocs

**Review by caitlinjee — APPROVED** [review]

**Review by jiaqi20 — APPROVED** [review]
lgtm

**Review by oliviajohansen — APPROVED** [review]
lgtm

**Review by TianYong-Goh — APPROVED** [review]
lgtm

**Pull request merged**